stetho/src/main/java/com/facebook/stetho/common/StringUtil.java

```diff
@@ -29,4 +29,16 @@ public static String removePrefix(String string, String prefix) {
       return string;
     }
   }
+
+  public static String remove(String string, char target) {
```

Comment:
Let's rename this to `removeAll`. My suggestion to call it `remove` was almost right, but it implies that it behaves like a regular `Collection#remove` call which would only remove the 1st occurrence that is found. (I would've put up my own PR to change this, but I found some other things in this PR to fix up, so we can just roll it all together)